user/pages/04.Reference/10.resource-limits-quota-usage/docs.en.md

```diff
@@ -1,19 +1,49 @@
 ---
-title: 'REST API for Quota Information'
+title: 'Resource Limits, REST API for Quota and Usage'
 published: true
 date: '26-07-2021 16:25'
 taxonomy:
     category:
         - docs
 ---
 
-## Introduction
+## Reasoning behind resource limits
 
-For many resource types like instances, VCPUs, volume storage you can check the limits and the current usage in the dashboard (Horizon) or use the standard OpenStack APIs to collect the data in some automation. But information about quota and usage of object storage (S3) is not available using standard APIs.
+The SysEleven Stack imposes resource limits per region on customer projects. This may sound a little bit bossy, given the fact that our customers have access to plans where they pay on-demand what they use. Couldn't we waive all restrictions and let customers use what they want? For the time being, we decided to stick with quota limits, because they server for the following purposes:
+
+### Infrastructure protection
+
+Quotas prevent individual customers from depleting all resources and thus prevents the platform from possible damage and ensures that other customers remain able to use their fair share.
```

Comment:
Sounds to me like a background article, and not reference guide – "Erklärungen und Hintergrundinformationen zu unseren Diensten."
What do you think, could this be a separate background article?

Reply:
Indeed, there is some background around the reference, but I doubt it is enough to make a background article out of it. Any Idea, how to solve this dilemma?

Reply:
I am ok if you dont want to make it a background article; IMO though lines 10 to 37 + some information on why customer should implement monitoring e.g. using the syseleven exporter and the rest api would justify a background article. In the REST API reference docs there could be a backlink to the background article, and vice versa.

Reply:
```suggestion
Quotas prevent individual customers from depleting all resources and thus prevent the platform from possible damage and ensure that other customers remain able to use their fair share.
```

Reply:
Okay, finally got to apply the desired changes. Let's get that merged.